global.json

```diff
@@ -2,5 +2,8 @@
     "sdk": {
         "version": "6.0.100",
         "rollForward": "latestMinor"
+    },
+    "msbuild-sdks": {
+        "Microsoft.Quantum.Sdk": "0.24.41605-pull"
```

Comment:
I didn't meet such a format of the version numbers before (see the global.json in Q#RT repo).